File: dotnet/videostore/NewReleaseMovie.cs

namespace videostore
{
    class NewReleaseMovie : Movie
    {
        public NewReleaseMovie(string title, int priceCode) : base(title, priceCode)
        {
        }

        public override double Price(int daysRented)
        {
            return daysRented*3;
        }

        public override int FrequentRenterPoints(int daysRented)
        {
            return (daysRented > 1) ? 2 : 1;
        }
    }
}

File: java/videostore/src/NewReleaseMovie.java
class NewReleaseMovie extends Movie {
  NewReleaseMovie(String title) {
    super(title);
  }

  @Override
  double determineAmount(int daysRented) {
    return daysRented * 3;
  }

  @Override
  int determineFrequentRenterPoints(int daysRented) {
    return (daysRented > 1) ? 2 : 1;
  }
}
